tests/PhpPact/Consumer/InteractionBuilderTest.php

```diff
@@ -61,13 +36,13 @@ public function testSimpleGet()
             ->addHeader('Content-Type', 'application/json');
 
         $builder = new InteractionBuilder(new MockServerEnvConfig());
-        $result  = $builder
-            ->given('A test request.')
+        $builder
+            ->given('A test request.', ['key' => 'value'])
             ->uponReceiving('A test response.')
             ->with($request)
             ->willRespondWith($response);
 
-        $this->assertTrue($result);
+        $this->assertFalse($builder->verify());
```

Comment:
Like in my previous comment, I think it is good practice to assert values assigned to meaningful variable instead of mixing up Act and Assert sections together.

Reply:
Created variable for it.